eox_nelp/stats/templates/tenant_stats.html

```diff
@@ -8,11 +8,11 @@
 <body>
     <div id="tenant-stats"></div>
     <script>
-      var showVideos = "true" === "${videos}";
-      var showCourses = "true" === "${courses}";
-      var showProblems = "true" === "${problems}";
-      var showLearners = "true" === "${learners}";
-      var showInstructors = "true" === "${instructors}";
+      var showVideos = "false" === "${hideVideos}";
```

Comment:
I like the refactoring of the query parameters in the theme. The only thing is the logic, so In control, we have to thing in hidden items. So I have to use a blacklist logic.
Mainly, I am looking at this view as the feedback API endpoint for example.  We have a boolean parameter `public`.
If we set it true it means public but if I set it false not public or "private, hide-public". The thing is that we want to manage this logic of blacklist? 
Also, I have a mismatch of the query param, this is a django view, query params in snake case, but ups the component use Camelcase.

Reply:
please stop using racist terms, but  anyway that is just a semantic thing if the label say hide or show the user should be smart enough to select the right option